kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala

```diff
@@ -80,7 +82,7 @@ abstract class AbstractOperation(opType: OperationType, session: Session)
       case _ =>
     }
     info(s"Processing ${session.user}'s query[$statementId]: ${state.name} -> ${newState.name}," +
-      s" statement: $statement$timeCost")
+      s" statement: $redactedStatement$timeCost")
```

Comment:
not related this PR, shall we and `, ` between `$redactedStatement` and `$timeCost`  ?

Reply:
@ulysses-you comma is already added from here: https://github.com/apache/incubator-kyuubi/blob/e4f5ba7010dd497f40d51f5677b16d3cf7b882d4/kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala#L81

Reply:
ah I see, thank you !